README.rst

```diff
@@ -41,6 +41,15 @@ Optional:
 
 -  docutils (for documentation)
 
+Platform-Specific Instructions
+""""""""""""""""""""""""""""""
+
+The build system will try to pull in as many missing dependencies as it can into the build environment. However, this has a few caveats.
```

Comment:
I'm not sure but maybe it's worth mentioning the mechanism used for this (meson subprojects), as well as the fact that you can force it to ignore system libraries or refuse to download subprojects (meson's `--wrap-mode` option).